### ISSUE TITLE
Is the training normal?

### ISSUE THREAD

**Issue by CHDTevior** [yifan123/flow_grpo#151]
I am using a single-node dual-card H200 for training. I modified the configuration file as follows:
`
    gpu_number=2
    config = compressibility()
    config.dataset = os.path.join(os.getcwd(), "dataset/pickscore")

    # flux
    config.pretrained.model = "black-forest-labs/FLUX.1-dev"
    config.sample.num_steps = 6
    config.sample.eval_num_steps = 28
    config.sample.guidance_scale = 3.5

    config.resolution = 512
    config.sample.train_batch_size = 3
    config.sample.num_image_per_prompt = 6 #24
    config.sample.num_batches_per_epoch = int(48/(gpu_number*config.sample.train_batch_size/config.sample.num_image_per_prompt)) # 48
    assert config.sample.num_batches_per_epoch % 2 == 0, "Please set config.sample.num_batches_per_epoch to an even number! This ensures that config.train.gradient_accumulation_steps = config.sample.num_batches_per_epoch / 2, so that gradients are updated twice per epoch."
    config.sample.test_batch_size = 16 # This bs is a special design, the test set has a total of 2048, to make gpu_num*bs*n as close as possible to 2048, because when the number of samples cannot be divided evenly by the number of cards, multi-card will fill the last batch to ensure each card has the same number of samples, affecting gradient synchronization.

    config.train.batch_size = config.sample.train_batch_size
    config.train.gradient_accumulation_steps = config.sample.num_batches_per_epoch//2
    config.train.num_inner_epochs = 1
    config.train.timestep_fraction = 0.99
    config.train.beta = 0
    config.sample.global_std = True
    config.sample.same_latent = False
    config.train.ema = True
    config.sample.noise_level = 0.9
    config.save_freq = 10 # epoch
    config.eval_freq = 10
    config.save_dir = 'logs/pure_pickscore/flux-group24-8gpu_save10'
    config.reward_fn = {
        "pickscore": 0.5,
        "ocr": 0.2,
        "aesthetic": 0.3
    }
    
    config.prompt_fn = "general_ocr"

    config.per_prompt_stat_tracking = True
`
This is the reward change curve when I trained the OCR task on it：

<img width="1197" height="438" alt="Image" src="https://github.com/user-attachments/assets/7f7577f7-0317-4573-980d-7e84c0e867fc" />

<img width="1223" height="444" alt="Image" src="https://github.com/user-attachments/assets/24d96360-ae46-488b-bb03-65e3fd226e2c" />

<img width="1207" height="441" alt="Image" src="https://github.com/user-attachments/assets/bb78e1b4-0945-42ae-8b56-4f9d7d03286b" />

For the generated images, I found that they shifted somewhat in the direction I was hoping for, but only within the first ~200 steps of training，and the improvement of pickscore and OCR is not synchronized：

<img width="3004" height="713" alt="Image" src="https://github.com/user-attachments/assets/27a1c745-ab2d-407b-8764-960be5aaa731" />

Moreover, the results of different texts all converged to a very similar image (the OCR task goal was indeed achieved). This is very similar to the result when I trained with aesthetic rewards alone, which was a portrait of an old man with the same color style. I would like to ask if this is normal?

<img width="2920" height="583" alt="Image" src="https://github.com/user-attachments/assets/61e60f49-f674-40c7-ad1c-a338cc7fa72e" />
